itest/rust/build.rs

```diff
@@ -179,12 +243,15 @@ fn main() {
     let rust_file = rust_output_dir.join("gen_ffi.rs");
     let gdscript_template = godot_input_dir.join("GenFfiTests.template.gd");
     let gdscript_file = godot_output_dir.join("GenFfiTests.gd");
+    let gdscript_property_template_file = godot_output_dir.join("GenPropertyTemplate.gd");
```

Comment:
This could have the same naming convention as above: `"GenPropertyTests.template.gd`.

Also class name should maybe be the same as `GenPropertyTests`.